harness/determined/deploy/gcp/terraform/modules/compute/main.tf

```diff
@@ -8,11 +8,12 @@ resource "google_compute_instance" "master_instance" {
   name = "det-master-${var.unique_id}-${var.det_version_key}"
```

Comment:
What is the plan for these changes?  They look suspiciously like development-only changes.